admin_guide/ipsec.adoc

```diff
@@ -58,14 +58,14 @@ made aware of the change by removing the SDN interface and restarting the
 +
 ----
 # systemctl stop atomic-openshift-node
-# ovs-vsctl del-br br0
+# oc exec ovs-vsctl del-br br0
```

Comment:
Correct one is  oc exec <pod_name> -- ovs-vsctl del-br br0

Reply:
Fixed